renderdoc/driver/vulkan/wrappers/vk_cmd_funcs.cpp

```diff
@@ -7570,6 +7571,379 @@ void WrappedVulkan::vkCmdEndRendering(VkCommandBuffer commandBuffer)
   }
 }
 
+template <typename SerialiserType>
+bool WrappedVulkan::Serialise_vkCmdBuildAccelerationStructuresIndirectKHR(
+    SerialiserType &ser, VkCommandBuffer commandBuffer, uint32_t infoCount,
+    const VkAccelerationStructureBuildGeometryInfoKHR *pInfos,
+    const VkDeviceAddress *pIndirectDeviceAddresses, const uint32_t *pIndirectStrides,
+    const uint32_t *const *ppMaxPrimitiveCounts)
+{
+  SERIALISE_ELEMENT(commandBuffer);
+  SERIALISE_ELEMENT(infoCount).Important();
+  SERIALISE_ELEMENT_ARRAY(pInfos, infoCount);
+  SERIALISE_ELEMENT_ARRAY(pIndirectDeviceAddresses, infoCount);
+  SERIALISE_ELEMENT_ARRAY(pIndirectStrides, infoCount);
+
+  // Convert the array of arrays for easier serialisation
+  {
+    rdcarray<rdcarray<uint32_t>> maxPrimitives;
+    maxPrimitives.reserve(infoCount);
+
+    for(uint32_t i = 0; i < infoCount; ++i)
+    {
+      maxPrimitives.push_back({});
+      auto &rangeInfo = maxPrimitives.back();
+
+      rangeInfo.resize(pInfos[i].geometryCount);
+      memcpy(rangeInfo.data(), ppMaxPrimitiveCounts[i], rangeInfo.size() * sizeof(uint32_t));
```

Comment:
This could maybe be simplified to just `rangeInfo.append(ppMaxPrimitiveCounts[i], pInfos[i].geometryCount);` ?